.gitignore

```diff
@@ -69,9 +69,26 @@ CorsixTH/CorsixTH.sln
 *.class
 LevelEdit/bin/
 
+# Debugging related files
 CorsixTH/Lua/debug_script.lua
 CorsixTH/Lua/debugger.lua
+mobdebug.lua
 
 # For AnimView output
 VSPR-0.xml
 *export.log
+
+# CMake related files
```

Comment:
Since you added this header, the top five items in this file are all CMake related as well.